src/block_misc.rs

```diff
@@ -1053,7 +1053,7 @@ mod tests {
             ]
         ); // backward part  (5+6+7)
```

Comment:
Same comment as above mod the sum at the end

Reply:
Removed.